src/nupic/os/Regex.cpp

```diff
@@ -46,7 +47,14 @@ namespace nupic
       exactRegExp += re;
       if (re[re.length()-1] != '$') 
         exactRegExp += '$';
-      
+
+#if defined(NTA_OS_WINDOWS)
```

Comment:
Can you confirm that pcre does not work on Windows? If so, please file an issue for removing pcre when std::regex is supported and add a comment here linking to the issue. But I would prefer if we can just use pcre without the platform check.